### ISSUE TITLE
Grad_norm is Nan when resume training

### ISSUE THREAD

**Issue by Mingfeng-Wang** [open-mmlab/mmdetection#7189]
TorchVision: 0.10.1+cu111
OpenCV: 4.5.3
MMCV: 1.3.9
MMCV Compiler: GCC 7.5
MMCV CUDA Compiler: 11.1
MMDetection: 2.12.0+/2.14.0+
TORCH_VERSION=1.9.1

Train SparseRCNN_R50_8stages
Train on single GPU 3080ti. Each GPU has 4 images. Cumulate gradients for 4 iterations to backpropagate.
Nothing else has been changed compared to the "sparse_rcnn_r50_fpn_1x_coco.py"
When I resumed training, I found that grad_norm become NaN but the original training (training at one time) will not have any NaN value.

If the checkpoint file is needed, I am very happy to upload it.

**Comment by jshilong** [comment]
Maybe there are potential bugs in the ` Cumulate gradients hook`,  because I check the normal resume operation is no problem. we will check it asap

**Comment by lpz7** [comment]
I encountered the same problem, did you solve it?

**Comment by fengye1966** [comment]
samething happened on me, could you pls solve it?
